pkg/apis/serving/v1alpha1/kfservice_types.go

```diff
@@ -79,13 +83,21 @@ type SKLearnSpec struct {
 	RuntimeVersion string `json:"runtimeVersion,omitempty"`
 	// Defaults to requests and limits of 1CPU, 2Gb MEM.
 	Resources v1.ResourceRequirements `json:"resources,omitempty"`
+	// Mount a PersistentVolumeClaim for model
+	PersistentVolumeClaim *PersistentVolumeClaim `json:"persistentVolumeClaim,omitempty"`
```

Comment:
please validate PersistentVolumeClaim in validation webhook, for example `MountPath` should be a valid directory path

Reply:
The part has been removed in new solution, thanks.